.github/workflows/build-complete-samples.yml

```diff
@@ -830,7 +830,10 @@ jobs:
             - project_path: 'samples/tab-ui-templates/ts'
               name: 'tab-ui-templates'
               version: '18.x'
-            
+
+            - project_path: 'samples/tab-navbar-menu/ts'
+              name: 'tab-navbar-menu'
+              version: '16.x'
```

Comment:
Deployment pipeline is also missing here.

Reply:
Deployment pipeline is not required.